hngprojects/hng_boilerplate_java_web
# [Feat]: Add CRUD operations for products under an organization

**Changes**:
- Added methods in the `ProductController` for creating, deleting, and editing products.
- Implemented authorization checks to ensure only admins can perform these operations.
- Added exception handling for unauthorized access and product not found scenarios.
- Updated the Swagger documentation with operation summaries for each endpoint.

**How to Test**:
- Use Postman or similar tools to send HTTP requests to the implemented endpoints.
- Verify that only users with the admin role can create, delete, or edit products.
- Check that the proper error messages and status codes are returned for unauthorized actions or missing products.

**Related Issues**: 
[658](https://github.com/hngprojects/hng_boilerplate_java_web/issues/658)
